pyproject.toml

```diff
@@ -1,44 +1,148 @@
-[build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
-
-[project]
-authors = [
-    {name = "Saritasa", email = "pypi@saritasa.com"},
+[tool.poetry]
+name = "django-import-export-extensions"
+version = "0.4.1"
+description = "Extend functionality of `django-import-export`"
+license = "MIT"
+authors = ["Saritasa <pypi@saritasa.com>"]
+maintainers = [
+    "Nikita Azanov <nikita.azanov@saritasa.com>",
+    "Vladislav Romaschenko <vladislav.romaschenko@saritasa.com>",
+    "Andrey Otto <andrey.otto@saritasa.com>",
+    "Stanislav Khlud <stanislav.khlud@saritasa.com>",
+]
+readme = "README.rst"
+repository = "https://github.com/saritasa-nest/django-import-export-extensions"
+documentation = "https://django-import-export-extensions.readthedocs.io"
+keywords = [
+    "python",
+    "json",
+    "django",
+    "csv",
+    "xlsx",
+    "celery",
+    "import_export",
+    "django_import_export_extensions"
 ]
-requires-python = ">=3.9"
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
     "Framework :: Django",
     "Framework :: Django :: 3.2",
     "Framework :: Django :: 4.0",
     "Framework :: Django :: 4.1",
     "Framework :: Django :: 4.2",
```

Comment:
Please add django 5.0

Reply:
Or better check if poetry could set it on build. Use this to read whl files https://filext.com/file-extension/WHL
![image](https://github.com/saritasa-nest/django-import-export-extensions/assets/43534015/a563d3bb-1735-4c29-ad31-2e3134a7b04e)